Set the actual version of www/qt6-webengine, otherwise the version of qt6-base,
which is present in the ports tree, would be assumed and used during
compilation.

--- CMakeLists.txt.orig	2023-07-07 17:40:32 UTC
+++ CMakeLists.txt
@@ -15,7 +15,7 @@ project(QtWebEngine
 find_package(Qt6 6.2 CONFIG REQUIRED COMPONENTS BuildInternals Core)
 
 project(QtWebEngine
-    VERSION ${Qt6Core_VERSION}
+    VERSION %%DISTVERSION%%
     DESCRIPTION "QtWebEngine and QtPdf modules"
     HOMEPAGE_URL "https://qt.io/"
     LANGUAGES CXX C
